ion-schema-tests-runner/tests/ion-schema-tests-1-0.rs

```diff
@@ -9,8 +9,6 @@ ion_schema_tests!(
         "constraints::occurs::invalid::occurs_for_a_field_must_be_a_positive_integer_or_a_non_empty__non_negative_integer_range__7_",
         "constraints::occurs::invalid::occurs_must_be_a_positive_integer_or_non_empty__non_negative_integer_range__06_",
         "constraints::occurs::invalid::occurs_range_must_be_a_valid__satisfiable_range__1_",
-        "constraints::scale::invalid::scale_must_be_an_integer_or_range__10_",
-        "constraints::scale::invalid::scale_must_be_an_integer_or_range__11_",
```

Comment:
Just curious—is this something that is fixed in this PR or are these just tests that were incorrectly being skipped?

Reply:
No this was due to an issue in `Range`s. All the changes in the `isl_range` module are related to that.   

> fixes an issue with range construction which had a missing type check for range construction (https://github.com/amazon-ion/ion-schema-rust/commit/95e97c973dfa06083227bfabda4813cceddaaefe)